### PR TITLE
Pull Request for the project Positioning and Floating Elements

### PR DESCRIPTION
**Pull request by jose-Abel**
Hello,

This is the code review request for the 3rd project on HTML and CSS section of the Microverse curriculum "Positioning and Floating Elements", hoping to get amazing feedback!

Happy coding!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,37 @@
...
+- CSS
+- font-awesome
+- google api's font
+
+## Authors
+### Author1
+- 👤GitHub: https://github.com/danielufeli
+
+## Author2
+- 👤GitHub: https://github.com/jose-Abel
```
Comment (krys2fa) [inline README.md]: - [x] [MANDATORY] Your README.md file should always include a link to a live version of your project for easy access by reviewers, potential collaborators and recruiters. You can host your projects on Github pages, Githack or Netlify.

- [x] [OPTIONAL] You can make your README.md file look professional here by writing your handles in markdown like this:
```
Github: [Daniel Ufeli](https://github.com/danielufeli)
```

**Review by krys2fa — CHANGES_REQUESTED** [review]
### Good job on the project @jose-Abel and @danielufeli 👏 
You made good use of semantic HTML and utilized CSS best practices 🖖  

### Status: Changes Required ♻️ 
Kindly work on the following issues to get this project approved. Don't forget to request a code review after implementing the changes.

- [x] [MANDATORY] The navigation bar should have a fixed position (i.e. it sticks to the top part of the screen).
- [x] [MANDATORY] The footer should have a fixed position (i.e. it sticks to the bottom part of the screen).
- [x] [MANDATORY] The width of the image, caption and article text should be the same as seen on the original page.    

Original Page          |  Your Page
:-------------------------:|:-------------------------:
 Screen Shot 2020-07-17 at 10 25 54 PM|Screen Shot 2020-07-17 at 10 27 03 PM


- [x] [MANDATORY] The text under the video is not placed right. Kindly fix this.

Original Page          |  Your Page
:-------------------------:|:-------------------------:
Screen Shot 2020-07-17 at 10 22 09 PM|  Screen Shot 2020-07-17 at 10 21 11 PM

- [x] [MANDATORY] 'The More in Space and Astronomy' section is off. The title does not also show up. Kindly fix this.

Original Page          |  Your Page
:-------------------------:|:-------------------------:
 Screen Shot 2020-07-17 at 10 33 21 PM|Screen Shot 2020-07-17 at 10 32 41 PM

- [x] [MANDATORY] The images in 'The Editor Picks' section are not aligned. Kindly fix this.

Original Page          |  Your Page
:-------------------------:|:-------------------------:
 Screen Shot 2020-07-17 at 10 37 39 PM|Screen Shot 2020-07-17 at 10 35 02 PM

File: README.md (hunk 1 above)
Comment (jose-Abel) [reply README.md]: Hello, 
Thanks for the feedback! We address all the mistakes that you indicate to us and we believe the code is ready for another code review. Appreciate your feedback!

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: 

#### The page looks awesome, the code clean and we are almost there. 

- [x] The footer does not have a fixed position

- [x]  The line in the top of the editor's pick is a bit off

| Yours | Original |
|:---:|:---:|
| image | image |

### Do these two things and submit for a new review.  The project looks really great.

Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: 

#### You just have a few linter errors now. Kindly fix that and submit again. Like I said before, the project looks really good and you implemented everything that was requested :clap: 

- [x] linter errors

Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Review by expjazz — APPROVED** [review]
## Status:  Project Approved :100: 

#### Good luck on the next projects!

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Pull request merged**